frontend/server/src/main/java/com/amazonaws/ml/mms/http/ManagementRequestHandler.java

```diff
@@ -167,6 +168,10 @@ private void handleRegisterModel(ChannelHandlerContext ctx, QueryStringDecoder d
         int initialWorkers = NettyUtils.getIntParameter(decoder, "initial_workers", 0);
         boolean synchronous =
                 Boolean.parseBoolean(NettyUtils.getParameter(decoder, "synchronous", null));
+        String responseTimeout = NettyUtils.getParameter(decoder, "response_timeout", null);
```

Comment:
We need update OpenApi document if we make this api change.

Reply:
Will make this change.